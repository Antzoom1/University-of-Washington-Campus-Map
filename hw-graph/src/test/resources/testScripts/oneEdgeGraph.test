# Tests that a singular edge can be added to a graph.

# Create a graph
CreateGraph threeNodesGraph

# Add three nodes
AddNode threeNodesGraph 1
AddNode threeNodesGraph 2
AddNode threeNodesGraph 3

# Add one edge
AddEdge threeNodesGraph 1 2 e1

# Print the nodes in the graph and the outgoing edges from 1
ListNodes threeNodesGraph
ListChildren threeNodesGraph 1