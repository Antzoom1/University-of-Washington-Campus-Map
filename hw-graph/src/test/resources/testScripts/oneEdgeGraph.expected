# Tests that a singular edge can be added to a graph.

# Create a graph
created graph threeNodesGraph

# Add three nodes
added node 1 to threeNodesGraph
added node 2 to threeNodesGraph
added node 3 to threeNodesGraph

# Add one edge
added edge e1 from 1 to 2 in threeNodesGraph

# Print the nodes in the graph and the outgoing edges from 1
threeNodesGraph contains: 1 2 3
the children of 1 in threeNodesGraph are: 2(e1)